README.md

```diff
@@ -43,10 +103,11 @@ We're always looking for new contributions! You can add your Prefect 2.0 recipe
 >git checkout feat/recipe-name
 >```
 >5. Add your code under the appropriate category, making sure it is reproducible and easy to understand.
->6. Commit and push the code to your remote branch.
->7. Create a PR 🤌 
```

Comment:
Should these be prefixed with `>` or just numbered?
![image](https://user-images.githubusercontent.com/15331990/185481585-6ee7631d-ff98-4a4a-83f0-cc5a3e723e0b.png)